tools/sggcov2lcov/tools.go

```diff
@@ -0,0 +1,70 @@
+package sggcov2lcov
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"runtime"
+	"strings"
+
+	"go.einride.tech/sage/sg"
+	"go.einride.tech/sage/sgtool"
+)
+
+const (
+	name    = "gcov2lcov"
+	version = "1.0.6"
+)
+
+func Command(ctx context.Context, args ...string) *exec.Cmd {
+	sg.Deps(ctx, PrepareCommand)
+	coverFile := sg.FromBuildDir("go", "coverage", "go-test.txt")
+	lcovFile := sg.FromBuildDir("go", "coverage", "go-test.lcov")
```

Comment:
The convention in Sage is generally to keep the command as clean as possible running nothing but the tool and leaving any conventions with the tool to more specific commands. Can we leave these out of this command?

Reply:
Good point, I have moved these to be arguments instead.

Reply:
The command  target should be a clean target and run nothing else but the command and given args. No assumptions about any flags should be made in here.

Either we should use the infile and outfile arguments locally in each project or break it out to a separate target